test/node/nocks/nock.js

```diff
@@ -74,8 +80,14 @@ get
     })
     .reply(200, {
         error: {
-            name: 'InvalidRequest',
-            message: 'Invalid request information entered'
+            code: -32000,
+            name: "JSONRPCError",
+            message: "The method 'Geet' could not be found. Verify the method name and ensure all method parameters are included.",
```

Comment:
The method "Get"

Reply:
Hard to tell from the diff (and maybe by the method name choosen) but this is intentional. The method called is 'Geet'.